lib/db.js

```diff
@@ -51,7 +51,6 @@ export default {
         if (
           key === "id" ||
           key === "eventID;year" ||
-          key === "createdAt" ||
```

Comment:
What happens if we don't remove this?

Reply:
It doesn't save a createdAt value in the DB